pkg/ingester/ingester.go

```diff
@@ -407,7 +415,17 @@ func (i *Ingester) starting(ctx context.Context) error {
 		servs = append(servs, closeIdleService)
 	}
 
-	var err error
+	marker, err := i.shutdownMarker.Exists()
```

Comment:
Nit: I'd do `s/marker/shutdownMarker/`, even though it's not far away, but still this is the `starting()` function, with a wider context.